### ISSUE TITLE
Won't run

### ISSUE THREAD

**Issue by RoLYroLLs** [marcsstevenson/AzureFunctionsWithUserSecretsTesting#1]
Hey, I tried your example. I don't see a local.settings.json file so I added one too.

Also, when I run the project, I get this (see image). I'm not too familiar so I'm not sure how to fix it.

![image](https://github.com/user-attachments/assets/f421a7ff-ff3d-4ac3-8c52-04106710a56b)

Let me know.

**Comment by RoLYroLLs** [comment]
My fault, forgot to add the following to the local.settings.json

```
{
  ...
  "Values": {
    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
  }
  ...
}
```

**Comment by marcsstevenson** [comment]
Hiya. Yes, that error results in missing settings values. I have just pushed my local.settings.json (which by convention is git ignored).

**Comment by RoLYroLLs** [comment]
I think I ran into a 'gotcha'. One thing I forgot to mention on my SO comment is I'm trying to access to config from within the `Program.cs`. Maybe that's why? I'll add a PR with my changes.

**Comment by marcsstevenson** [comment]
With this?

`config.AddJsonFile("local.settings.json");`

That shouldn't be required as the HostBuilder will look for a local.settings.json by convention.

**Comment by RoLYroLLs** [comment]
Yeah, lemme test on your Example, maybe something on my app not working instead. Give me a few minutes. :)

**Comment by RoLYroLLs** [comment]
Ok So here's what I'm trying to do (maybe I'm doing it wrong) which is why I need that line `config.AddJsonFile("local.settings.json");`. I noticed I had to add `Values:` to my `local.settings.json` file, but then these cannot be called the way you call them in `ConfigGoGoFunction.cs`, which is now making me thing I'm going this incorrectly.

I'll send a PR in a bit and here's a screenshot of the values I get.

![image](https://github.com/user-attachments/assets/33301b9e-f976-4683-b702-e4273ec6c972)
